debian/rustc.install

```diff
@@ -4,10 +4,10 @@
 /usr/bin/rustc
 /usr/bin/rustdoc
 /usr/bin/rustfmt
-/usr/lib/libLLVM-13-rust-1.56.0-stable.so
-/usr/lib/librustc_driver-d767902d1c6d74c6.so
-/usr/lib/libstd-8142ba460901c3ec.so
-/usr/lib/libtest-dfcbb96fdb281dd3.so
+/usr/lib/libLLVM-13-rust-1.56.1-stable.so
```

Comment:
Is there any reason this couldn't also use an `*`, so it doesn't have to be updated for new versions of llvm and Rust?

Something like `/usr/lib/libLLVM-*-rust-*-stable.so`.